packages/forklift-console-plugin/src/modules/Providers/views/create/components/ProviderCreateForm.tsx

```diff
@@ -103,7 +124,20 @@ export const ProvidersCreateForm: React.FC<ProvidersCreateFormProps> = ({
         <EditProviderSectionHeading text={t('Provider details')} />
 
         <Form isWidthLimited className="forklift-section-secret-edit">
-          <FormGroupWithHelpText label={t('Select provider type')} isRequired fieldId="type">
+          <ProjectNameSelect
+            value={projectName}
+            options={projects.map((project) => ({
+              value: project.metadata?.name,
+              content: project.metadata?.name,
+            }))}
+            isDisabled={!projectsLoaded || !projects.length}
+            onSelect={onProjectNameChange}
+            popoverHelpContent={
+              <ForkliftTrans>The project that your provider will be created in.</ForkliftTrans>
+            }
+          />
+
+          <FormGroupWithHelpText label={t('Provider type')} isRequired fieldId="type">
```

Comment:
hmm, this was wrong before, but we can fix it now, we don't select the provider type, we select the actual provider, e.g. we do not select the type "vmware" vs "ovirt" we delect "vmware server 1" vs "vmware server 2" both same type but different providers